diff -Nur soup-0.7.4.orig/src/libwsdl/Makefile.am soup-0.7.4.chng/src/libwsdl/Makefile.am
--- soup-0.7.4.orig/src/libwsdl/Makefile.am	Fri Apr  5 20:46:30 2002
+++ soup-0.7.4.chng/src/libwsdl/Makefile.am	Wed Nov 13 21:06:28 2002
@@ -49,12 +49,6 @@
 noinst_LIBRARIES = libwsdl-build.a
 
 libwsdl_build_a_SOURCES =	\
-	wsdl-typecodes.c	\
-	wsdl-typecodes.h	\
-	wsdl-schema.c		\
-	wsdl-schema.h		\
-	wsdl-schema-glib.c	\
-	wsdl-schema-glib.h	\
 	wsdl-typecodes-c.c	\
 	wsdl-typecodes-c.h
 
diff -Nur soup-0.7.4.orig/src/soup-wsdl/Makefile.am soup-0.7.4.chng/src/soup-wsdl/Makefile.am
--- soup-0.7.4.orig/src/soup-wsdl/Makefile.am	Wed Apr 10 16:33:19 2002
+++ soup-0.7.4.chng/src/soup-wsdl/Makefile.am	Wed Nov 13 21:04:32 2002
@@ -17,6 +17,7 @@
 	$(POPT_LIBS)					\
 	$(XML_LIBS)					\
 	$(top_builddir)/src/libsoup/libsoup.la		\
+	$(top_builddir)/src/libwsdl/libwsdl.la		\
 	$(top_builddir)/src/libwsdl/libwsdl-build.a
 
 soup_wsdl_SOURCES = 		\
